src/main/drivers/bus_i2c_stm32f30x.c

```diff
@@ -129,6 +129,12 @@ uint16_t i2cGetErrorCounter(void)
 
 bool i2cWrite(I2CDevice device, uint8_t addr_, uint8_t reg, uint8_t data)
 {
+    return i2cWriteBuffer(device, addr_, reg, 1, &data);
+}
+
+
+bool i2cWriteBuffer(I2CDevice device, uint8_t addr_, uint8_t reg, uint8_t len, uint8_t *data)
```

Comment:
IMO @martinbudden worked on buffer write functions recently and maybe he used different argument order(data, len). This PR should be consistent ...